esphome/esp32-c3-mini--goe11-controll.yaml

```
@@ -581,29 +592,36 @@ sensor:
+        - if:
+            condition:
+              lambda: !lambda return x == id(card_zoe).state;
+            then:
+              - number.set:
+                  id: min_amps
+                  value: 8
+            else:
+              - number.set:
+                  id: min_amps
+                  value: ${min_amps}
+    #on_value_range: 
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)